### PR TITLE
Steals Bays modular armor, for later use

### PR DESCRIPTION
**Pull request by Anewbe**
Once Soak is fixed, it'll work for that too.

### CODE REVIEW

File: code/modules/clothing/under/accessories/armor.dm [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+	icon_state = "ecflag"
+
+/obj/item/clothing/accessory/armor/tag/sec
+	name = "\improper POLICE tag"
+	desc = "An armor tag with the word POLICE printed in silver lettering on it."
+	icon_state = "sectag"
+
+/obj/item/clothing/accessory/armor/tag/com
+	name = "\improper SCG tag"
+	desc = "An armor tag with the words SOL CENTRAL GOVERNMENT printed in gold lettering on it."
```
Comment (elgeonmb) [inline code/modules/clothing/under/accessories/armor.dm]: obviously bayfluff is bay

Comment (Anewbe) [reply code/modules/clothing/under/accessories/armor.dm]: Hidden bayfluff is hidden, none of these spawn anywhere.

File: code/__defines/items_clothing.dm [hunk 2]
```diff
@@ -22,6 +22,19 @@
 #define SLOT_TIE        0x4000
 #define SLOT_HOLSTER	0x8000 //16th bit - higher than this will overflow
 
+#define ACCESSORY_SLOT_UTILITY  "Utility"
+#define ACCESSORY_SLOT_ARMBAND  "Armband"
+#define ACCESSORY_SLOT_RANK     "Rank"
+#define ACCESSORY_SLOT_DEPT		"Department"
```
Comment (Belsima) [inline code/__defines/items_clothing.dm]: This one seems indented further than the rest.

File: code/modules/clothing/under/accessories/armor.dm [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+	name = "heavy leg guards"
+	desc = "A pair of heavily armored leg pads in red-trimmed black. Attaches to a plate carrier."
+	icon_state = "legguards_merc"
+	armor = list(melee = 60, bullet = 60, laser = 60, energy = 40, bomb = 40, bio = 0, rad = 0)
+
+
+//Decorative attachments
+/obj/item/clothing/accessory/armor/tag
+	name = "\improper SCG Flag"
+	desc = "An emblem depicting the Sol Central Government's flag."
```
Comment (Belsima) [inline code/modules/clothing/under/accessories/armor.dm]: Is this our flag or Bay's? Just asking so I know whether or not I should try and sprite a boi.

Comment (Anewbe) [reply code/modules/clothing/under/accessories/armor.dm]: I don't think those have sprites, but go ahead?

Comment (Belsima) [reply code/modules/clothing/under/accessories/armor.dm]: I mean `icon_state = "solflag"` - at school right now so I can't check, just asking if it should go on my to-do list.

Comment (Anewbe) [reply code/modules/clothing/under/accessories/armor.dm]: Up to you, I don't care.

**Comment by Mechoid** [comment]
Tested, no further discussion needed?

**Comment by Mechoid** [comment]
I have been informed it has indeed been tested

**Pull request merged**